README.md

```diff
@@ -15,106 +15,84 @@ The contract is designed to be universal and flexible:
+      ✔ cancel votes
+      ✔ cancel votes in reversed order
+      ✔ checking voting state
+      ✔ checking voting state after delegator re-voting
+      ✔ checking voting state after first voter re-voting
+      ✔ checking voting state after second voter re-voting
+      ✔ checking voting state after delegator vote canceling
+      ✔ checking voting state after first staker vote canceling
+      ✔ checking voting state after delegator recounting
+      ✔ checking voting state after first staker recounting
+      ✔ checking voting state after cross-delegations between voters
+      ✔ cancel votes via recounting
+      ✔ cancel votes via recounting gradually
+      ✔ cancel votes via recounting in reversed order
+      ✔ cancel votes via recounting gradually in reversed order
     checking votes for 2 self-voters and 1 delegation
-      ✓ cancel votes (205ms)
-      ✓ cancel votes in reversed order (212ms)
-      ✓ checking voting state (334ms)
-      ✓ checking voting state after delegator re-voting (481ms)
-      ✓ checking voting state after first voter re-voting (410ms)
-      ✓ checking voting state after second voter re-voting (440ms)
-      ✓ checking voting state after delegator vote canceling (247ms)
-      ✓ checking voting state after first staker vote canceling (231ms)
-      ✓ checking voting state after delegator recounting (360ms)
-      ✓ checking voting state after first staker recounting (311ms)
-      ✓ checking voting state after cross-delegations between voters (616ms)
-      ✓ cancel votes via recounting (203ms)
-      ✓ cancel votes via recounting gradually (242ms)
-      ✓ cancel votes via recounting in reversed order (221ms)
-      ✓ cancel votes via recounting gradually in reversed order (267ms)
+      ✔ cancel votes
+      ✔ cancel votes in reversed order
+      ✔ checking voting state
+      ✔ checking voting state after delegator re-voting
+      ✔ checking voting state after first voter re-voting
+      ✔ checking voting state after second voter re-voting
+      ✔ checking voting state after delegator vote canceling
+      ✔ checking voting state after first staker vote canceling
+      ✔ checking voting state after delegator recounting
+      ✔ checking voting state after first staker recounting
+      ✔ checking voting state after cross-delegations between voters
+      ✔ cancel votes via recounting
+      ✔ cancel votes via recounting gradually
+      ✔ cancel votes via recounting in reversed order
+      ✔ cancel votes via recounting gradually in reversed order
+
+  VotesBookKeeper
+    ✔ onVoted() should record two votes from one voter to two different proposals
+    ✔ onVoted() should revert when received too many votes from one voter
+    ✔ onVoteCanceled() removes vote
+    ✔ checking VotesBookKeeper proposals cap
+    ✔ checking VotesBookKeeper pruning outdated votes
+    ✔ checking VotesBookKeeper indexes
 
 
-  52 passing (1m)
+  62 passing (2s)
```

Comment:
Do we need the sample test output in the README? I would prefer to remove it, to avoid the need to update it.